components/service/firefox-accounts/src/main/java/mozilla/components/service/fxa/FirefoxAccount.kt

```diff
@@ -69,13 +69,7 @@ class FirefoxAccount internal constructor(private val inner: InternalFxAcct) : A
      */
     fun getProfile(ignoreCache: Boolean): Deferred<Profile> {
         return scope.async {
-            val internalProfile = inner.getProfile(ignoreCache)
-            Profile(
-                    uid = internalProfile.uid,
-                    email = internalProfile.email,
-                    avatar = internalProfile.avatar,
-                    displayName = internalProfile.displayName
-            )
+            inner.getProfile(ignoreCache).let { Profile.fromInternal(it) }
```

Comment:
nit (no need to change): you don't really need a separate scope here and could just write:
```kotlin
Profile.fromInternal(inner.getProfile(ignoreCache))
```
or
```kotlin
val internalProfile = inner.getProfile(ignoreCache)
Profile.fromInternal(internalProfile)
```